src/components/ResponsiveAppBar/ResponsiveAppBar.js

```diff
@@ -4,22 +4,27 @@ import AppBar from "@mui/material/AppBar";
 import Typography from "@mui/material/Typography";
 import Button from "@mui/material/Button";
 import { useNavigate } from "react-router";
+import "./ResponsiveAppBar.css";
 
 const ResponsiveAppBar = () => {
   const navigate = useNavigate();
   const handleClick = () => {
     navigate("/login");
   };
+  const handleNavigate = (route) => {
+    navigate(`${route}`);
```

Comment:
I like what you're doing - but it's over engineered! `route` is already a string - no need to wrap it in a string!

```suggestion
    navigate(route);
```

Reply:
fixed!